docs/reference/cli-reference.md

```diff
@@ -279,18 +289,27 @@ Create an identity.
 
 ### Synopsis
 
-Create a machine identity with NAME or a user identity with EMAIL.
+Create an identity.
 
-NAME must only contain alphanumeric characters ('a-z', 'A-Z', '0-9') or the
-special characters '-', '_', or '/' and has a maximum length of 256 characters.
```

Comment:
why remove the validity requirements?

Reply:
in discussion with @jmorganca and @FSHA for final changes, we wanted to keep descriptions short, since users should know valid formats. (ie. when you sign up for a random website, they don't really tell you email/username must be this regex, it just errors out and tells you why)

Reply:
That's fine for a user interface, but typically users would want to know exactly what's valid in the docs. Maybe it belongs in the API spec instead.

Reply:
@ssoroka +1 - for the CLI, we'd provide an error in the response should the user choose a bad name